private/buf/bufcli/errors.go

```diff
@@ -174,7 +175,9 @@ func wrapError(err error) error {
 			if dnsError := (&net.DNSError{}); errors.As(err, &dnsError) && dnsError.IsNotFound {
 				return fmt.Errorf(`%s Are you sure "%s" is a valid remote address?`, msg, dnsError.Name)
 			}
-
+			if tlsErr := (&tls.CertificateVerificationError{}); errors.As(err, &tlsErr) {
+				return tlsErr
```

Comment:
Just to be safe from connect messages error rephrases in the future, let's wrap it in a message?
```suggestion
				return fmt.Errorf("tls certificate verification: %w", tlsErr)
```

Reply:
Done.